packages/gestalt/src/Icon.js

```diff
@@ -104,10 +125,23 @@ export default function Icon({
   inline = false,
   size = 16,
 }: Props): Node {
+  let colorClass = null;
+  const colorName = semanticColors.includes(color) ? `${color}Icon` : color;
+  if (
+    colorName !== 'dark' &&
+    colorName !== 'error' &&
+    colorName !== 'light' &&
+    colorName !== 'subtle' &&
+    colorName !== 'success' &&
+    colorName !== 'warning'
```

Comment:
@ayeshakmaz , this seem to be missing. Any reason?

  | 'default'
  | 'inverse'
  | 'shopping'

Reply:
It's because they exist in Colors.css, so they don't create the Flow issue